### ISSUE TITLE
Add Travis support for main projects

### ISSUE THREAD

**Issue by jabrena** [ev3dev-lang-java/ev3dev-lang-java#575]
Add Travis support for all Stable projects.

**Stable projects:**
 
- [x] [OpenJDK for EV3](https://github.com/ev3dev-lang-java/openjdk-ev3): A custom OpenJDK JRI/JDK (9,10,11,12) build for EV3 
- [x]  [EV3Dev-lang-java](https://github.com/ev3dev-lang-java/ev3dev-lang-java): Low level interation with EV3Dev
- [x]  [lejos-commons](https://github.com/ev3dev-lang-java/lejos-commons): LeJOS interfaces & utilities
-  [Installer](https://github.com/ev3dev-lang-java/installer): A set of Bash scripts to automate some operations with your brick
- [x]  [Template project](https://github.com/ev3dev-lang-java/template_project_gradle): A Gradle project ready to test the project
- [x]  [Examples](https://github.com/ev3dev-lang-java/examples): A repository with several examples about the usage of this project

**Comment by jabrena** [comment]
https://travis-ci.org/ev3dev-lang-java/lejos-commons/builds/463262609

**Comment by jabrena** [comment]
It is necessary to clean Example project to unblock this issue

**Comment by jabrena** [comment]
For installer, it is necessary to create a new project to execute the all methods in a Container.
I will create a new issue only specific for that area.

**Comment by JakubVanek** [comment]
This is already in progress: https://github.com/JakubVanek/elj-install/tree/travis

**Comment by jabrena** [comment]
Added Travis support for `Installer`.
https://travis-ci.org/ev3dev-lang-java/installer/builds/465727367

Related issue:
https://github.com/ev3dev-lang-java/ev3dev-lang-java/issues/618
